packages/components/src/combobox-control/stories/index.js

```diff
@@ -287,3 +287,72 @@ _default.args = {
 	__next36pxDefaultSize: false,
 	allowReset: false,
 };
+
+const authors = [
+	{
+		name: 'Hermann P. Schnitzel',
+		age: 45,
+		country: 'Germany',
+	},
+	{
+		name: 'Shequondolisa Bivouac',
+		age: 43,
+		country: 'France',
+	},
+	{
+		name: 'Bodrum Salvador',
+		age: 42,
+		country: 'Spain',
+	},
+	{
+		name: 'Parsley Montana',
+		age: 48,
+		country: 'Germany',
+	},
+	{
+		name: 'Cabbage New York',
+		age: 44,
+		country: 'France',
+	},
+	{
+		name: 'Jake Weary',
+		age: 41,
+		country: 'United Kingdom',
+	},
+];
+
+const authorOptions = authors.map( ( { name, ...details } ) => ( {
+	value: name,
+	label: name,
+	...details,
+} ) );
+
+export const WithRenderOption = ( args ) => {
```

Comment:
I noticed that, in this Storybook example, even when there's no value set, the "X" button is shown in the right part of the input

![Uploading Screenshot 2022-08-02 at 13.34.06.png…]()

Reply:
Because `allowReset` prop is set to `true` by default.